pyproject.toml

```diff
@@ -71,33 +71,35 @@ all = [
     "astropy[ipython]",
     "astropy[jupyter]",
     # Install all remaining optional dependencies
-    "certifi",
-    "dask[array]>=2022.8.1",
-    "h5py",
-    "pyarrow>=10.0.1",
-    "beautifulsoup4",
-    "html5lib",
-    "bleach",
-    "pandas>=2.0",
-    "sortedcontainers",
-    "pytz",
-    "jplephem",
-    "mpmath",
-    "asdf-astropy>=0.3",
-    "bottleneck",
+    "certifi>=2022.6.15.1",
+    "dask[array]>=2022.5.1",
+    "h5py>=3.8.0", # 2023-01-23
+    "pyarrow>=10.0.1", # 2022-11-22
+    "beautifulsoup4>=4.9.3", # 2020-10-03, imposed by pandas==2.0
```

Comment:
It is good to record that this version requirement is imposed by another package, much more valuable than the release date.